index.html

```diff
@@ -0,0 +1,208 @@
+        <a class="navbar-brand mb-0 h1">My Page</a>
+        <a class="navbar-brand mb-0 h1">Logout</a>
+        
+    </nav>
+
+    <!-- Desktop and mobile nav -->
+
+    <nav class="navbar navbar-expand-md navbar-light bg-light">
+        <div class="container-fluid">
+        <a class="navbar-brand" href="#"><span class="text-warning">Tech</span>Con</a>
+        <button class="navbar-toggler collapsed d-flex d-md-none flex-column" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="toggler-icon top-bar"></span>
+            <span class="toggler-icon middle-bar"></span>
+            <span class="toggler-icon bottom-bar"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0 gap-2">
+            <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#home">Home</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="./about.html">About</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#program">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#join">Join</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#sponsor">Sponsor</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#news">News</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" id="campaign-link" href="#news">TC Campaign</a>
+            </li>
+            </ul>
+        </div>
+        </div>
+    </nav>
+
+    <!-- Event details page -->
+
+    <section class="event-section">
+
+        <div class="h-text">
+            <h4>"Hello World!!"</h4>
+            <h1>
+                TechCon<br/>Conference
+                Week
+              </h1>
+        </div>
+        <div class="p-text">
+           <p>TechCon Conference Week is a technology conference, an exhibition and a Technology Awards Ceremony.
+           </p>
+        </div>
+        <div class="h2-text">
+            <h4>CONNECT WITH US</h4>
+            <h4>27-28 December 2022</h4>
+            <h4>Virtual Event</h4>
+        </div>
+        
+    </section>
+
+    <!-- Main program card section mobile and desktop -->
+
+    <section class="card-section">
+        <div class="container">
+            <div class="row">
+                <div class="section-header text-center pb-2">
+                    <h2>Main Program</h2>
+                    <div class="guide_bar"></div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-md-6 col-lg mt-3">                  
```

Comment:
# Desktop size
- **It contains a medium line of text in bold letters and a smaller line of text.**
- **It contains five individual articles, each with an image, title, description, and a line of text at the bottom.**
- Your design
![Capture](https://user-images.githubusercontent.com/63972670/198668143-a85ea3ab-a3d6-4a88-90d8-6bec21417d18.PNG)
- Original design

![Capture1](https://user-images.githubusercontent.com/63972670/198668871-d10bdaac-6575-46fe-a099-37337bb83cb6.PNG)